plugins/org.eclipse.n4js.cli/src/org/eclipse/n4js/cli/N4jscOptions.java

```diff
@@ -56,12 +56,12 @@
 	/** Usage information template. */
 	public static final String USAGE_TEMPLATE = "Usage: n4jsc %s%s [OPTION(s)]";
 	/** Usage information. */
-	public static final String USAGE = getUsage(new N4jscOptions());
+	public static final String USAGE = getUsage(new ImplicitCompileOptions());
 
 	/** @return the usage string respecting the goal specified in the given options */
-	public static String getUsage(N4jscOptions options) {
-		String goal = options.options.isImplicitGoal() ? "[GOAL]" : options.getGoal().name();
-		String dir = options.options.getDir() == null ? "" : " [DIR]";
+	public static String getUsage(AbstractOptions options) {
+		String goal = options.isImplicitGoal() ? "[GOAL]" : options.getGoal().realName;
+		String dir = options.getDir() == null ? "" : " [DIR]";
```

Comment:
Considering the changes above, shouldn't this be `[ARG]` instead of `[DIR]`?

Also, iff the usage is printed for a specific goal, could `[ARG]` be replaced by the kind of that goal's argument? (i.e. `DIR`, `VERSION`, etc.)

Reply:
Probably related to the previous comment:

When doing `java -jar ./n4jsc.jar set-versions -h` on the command line, I get:
```
Usage: n4jsc set-versions [OPTION(s)]
 VERSION        : new version string to set for all n4js related dependencies
 --help (-h)    : prints help and exits. Define a goal for goal-specific help.
                  (default: false)
 --showSetup    : prints n4jsc setup (default: false)
 --verbose      : enables verbose output (default: false)
 --version (-v) : prints version and exits (default: false)
```
The line "Usage: ..." is missing the argument VERSION, i.e. should be
```
Usage: n4jsc set-versions VERSION [OPTION(s)]
```